ably/realtime/connection.py

```diff
@@ -45,41 +45,136 @@ class ProtocolMessageAction(IntEnum):
 
 
 class Connection(AsyncIOEventEmitter):
+    """Ably Realtime Connection
+
+    Enables the management of a connection to Ably
+
+    Attributes
+    ----------
```

Comment:
I'm wondering if we need to list all the attributes and methods up here. Will it work to just document each attribute and method individually?

Reply:
Ah, that makes sense, thanks for explaining 🙂